iceoryx_posh/test/moduletests/test_popo_publisher_port.cpp

```diff
@@ -370,33 +371,35 @@ TEST_F(PublisherPort_test, sendWhenSubscribedDeliversAChunk)
     EXPECT_TRUE(maybeSharedChunk.has_value());
     auto sharedChunk = maybeSharedChunk.value();
     auto dummySample = *reinterpret_cast<DummySample*>(sharedChunk.getPayload());
-    EXPECT_THAT(dummySample.dummy, Eq(17));
+    EXPECT_THAT(dummySample.dummy, Eq(17U));
 }
 
 TEST_F(PublisherPort_test, subscribeWithHistoryLikeTheARAField)
 {
-    iox::popo::PublisherPortData m_publisherPortDataHistory{
-        iox::capro::ServiceDescription("x", "y", "z"), "myApp", &m_memoryManager, 1u}; // history = 1
-    iox::popo::PublisherPortUser m_sutWithHistoryUseriSide{&m_publisherPortDataHistory};
-    iox::popo::PublisherPortRouDi m_sutWithHistoryRouDiSide{&m_publisherPortDataHistory};
+    iox::popo::PublisherOptions options;
+    options.historyCapacity = 1U;
+    iox::popo::PublisherPortData publisherPortDataHistory{
+        iox::capro::ServiceDescription("x", "y", "z"), "myApp", &m_memoryManager, options};
+    iox::popo::PublisherPortUser sutWithHistoryUseriSide{&publisherPortDataHistory};
+    iox::popo::PublisherPortRouDi sutWithHistoryRouDiSide{&publisherPortDataHistory};
     // do it the ara field like way
     // 1. publish a chunk to a not yet offered publisher
-    auto maybeChunkHeader = m_sutWithHistoryUseriSide.tryAllocateChunk(sizeof(DummySample));
+    auto maybeChunkHeader = sutWithHistoryUseriSide.tryAllocateChunk(sizeof(DummySample));
```

Comment:
Ahh, here the m_ is also no more good enough ;-)

Reply:
Personally I would encourage all to use the following rule: for structs or classes which do not have to encapsulate members and thus have them public, do not use the m_ prefix. It is just ugly for access then and I question the design in such a case.

Now, which classes are that? Simple: classes that do not really operate on their data AND for which any state set by the user, e.g. points in 3d space. But not something like a struct with two members a and b which always need to add up to some constant to be usable, the user can mess this up.

Reply:
@MatthiasKillat I will update the guidelines according your words